modules/datadict/README.md

```diff
@@ -0,0 +1,51 @@
+# Data Dictionary
+
+## Purpose
+
+The data dictionary module is used for viewing and modifying the
+LORIS data dictionary descriptions.
+
+## Scope
+
+The module displays and manages the data dictionary for things
+already stored in the LORIS `parameter_type` table. This table is
+generally autopopulated and provides a dictionary for imaging headers
+(populated by the imaging pipeline) and behavioural instrument
+(populated by the `lorisform_parser.php` script).
+
+It does not provide a way to enter new things into the data dictionary,
+or describe the data dictionary for arbitrary SQL tables which are
+not stored in the `parameter_type` table, such as instrument flags
```

Comment:
"for arbitrary SQL tables which are not stored in the `parameter_type` table" reads a bit funny. It reads like there are tables stored into the parameter_type table.

Shouldn't the word "SQL tables" be replaced by "SQL fields" or simply "fields"?

Reply:
Replace tables with fields here too.